### ISSUE TITLE
Stuttering when pressing keys + Suggestions

### ISSUE THREAD

**Issue by Alan-Jacob** [133spider/AQLite#49]
Issues: 
The game sometimes becomes very laggy (FPS) and starts freezing for .25 - 1 seconds any time I press a key. It's bad to the point where I can't properly time abilities because I cant see when they're off cooldown, and when I'm chatting, the game is just frozen until I stop typing.

Suggestions:
I think when you minimize the Travel Menu, you should still see the "Travel Next" button. It covers a lot of the screen and you dont really need to see the whole menu after you set it up, I feel.
Also, would it be possible to freeze other players characters at the entrance of rooms or something? Farming with a melee-range class is rough in rooms where people are spread apart, and freezing mobs doesn't help because they'll just respawn in their default spots again. If the players are frozen, the mobs can be unfrozen and they'd all go to the same spot.

**Comment by Alan-Jacob** [comment]
Another issue I just found; after being kicked from or switching servers, the travel menu breaks and only appears after a restart of the client.

**Comment by 133spider** [comment]
Yeah, the stuttering is something I've been trying to crack down on for awhile. The problem is mostly prevalent in ElectronJS since entering keyboard inputs on ElectronJS  is tied with your FPS.

The travel menu already has the keybind "T" which you can use while the menu is minimized

Instead of freezing other players, you can click on a mob's portrait and click "Freeze Monster" to have the monster stand in place.

Travel Menu breaking will be fixed in version 15
